process-engine/bpmn-studio
# ✨Add gateway and sequence flow help texts

## Changes

1. Add exclusive gateway section
2. Add condition usage help text for sequence flows

<img width="253" alt="Bildschirmfoto 2020-03-24 um 11 17 46" src="https://user-images.githubusercontent.com/17065920/77414397-3a276f00-6dc1-11ea-9f93-b6941cb73ac2.png">
<img width="512" alt="Bildschirmfoto 2020-03-24 um 11 18 17" src="https://user-images.githubusercontent.com/17065920/77414400-3b589c00-6dc1-11ea-843d-34c879c58c58.png">

## Issues

Closes #1993 

PR: #1995 

## How to test the changes

- create a diagram with an exclusive gateway with at least one outgoing sequence flow
- click on the gateway and see the information text at the property panel
- click on the outgoing sequence flow and on the information icon at the property panel `Flow` headline
- see the help text modal
